### PR TITLE
 feature: ensure the correct versions of the server-admin-ui and plugin-config are installed

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

**Comment by sbender9** [comment]
Now thinking maybe we should have some way to indicate that a package is an "internal package" for node server.  @rob42 is going to need a way to ignore these.

**Comment by tkurki** [comment]
Remove the signalk-webapp keyword and mount it explicitly?

**Comment by tkurki** [comment]
Can move to /admin path at the same time?

File: lib/serverroutes.js [hunk 1]
```diff
@@ -21,17 +21,22 @@ const path = require('path')
...
+      __dirname + '/../node_modules/@signalk/server-admin-ui/public'
+    )
+  )
+
   app.get('/', (req, res) => {
-    res.redirect('@signalk/server-admin-ui')
-  })
-  app.get('/admin', (req, res) => {
-    res.redirect('/@signalk/server-admin-ui')
+    res.redirect('/admin')
```
Comment (tkurki) [inline lib/serverroutes.js:40]: Should we add a redirect from /@signalk/server-admin-ui, reverse of what we had? There may be bookmarks and link here and there.

File: lib/config/config.js [hunk 2]
```diff
@@ -112,6 +117,20 @@ function load (app) {
   require('./production')(app)
 }
 
+function checkPackageVersion (name, pkg) {
+  const expected = pkg.dependencies[name]
+  const installed = require(`../../node_modules/${name}/package.json`)
+
+  if (!semver.satisfies(installed.version, expected)) {
+    console.error(
```
Comment (tkurki) [inline lib/config/config.js:125]: Is the only result of a failing check a line in the log? Should we have something very visible in the admin ui as well?

Comment (sbender9) [reply lib/config/config.js:125]: Below I `process.exit(1)`

Comment (tkurki) [reply lib/config/config.js:125]: Oops that you do.

File: lib/serverroutes.js (hunk 1 above)
Comment (sbender9) [reply lib/serverroutes.js:40]: done

**Pull request merged**